api/ProgressEvent.json

```diff
@@ -205,10 +205,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ProgressEvent/loaded",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": true
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": true
```

Comment:
WebView should be true.

Reply:
I found a way to test WebView, this is updated now.